screens/SignUpScreen.tsx

```diff
@@ -1,22 +1,22 @@
-import { StyleSheet } from 'react-native';
-import { View } from '../components/Themed';
+import React from 'react';
```

Comment:
do you need to import React? It should work without it

Reply:
I think you need React if you're going to use JSX. Not sure about that it .tsx files tho but that applies to .js files

Reply:
App.tsx doesn't have React import + you can try to delete it any .tsx file and it will work

Reply:
If you both agree, then i just delete it, if you don't see any reason in it.

Reply:
Quick updated: we have 17.0.1 React version so import React from 'react' is [no more needed](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

Reply:
Ok, good to know